### PR TITLE
[docs] - Update screenshots in tutorials

### PR DESCRIPTION
**Pull request by erinkcochran87**
### Summary & Motivation

This PR updates the screenshots in the SDA and ops/jobs tutorials and corrects some typos and formatting issues.

### CODE REVIEW

**Comment by shalabhc** [comment]
Thanks Erin! The screenshots still look like the older version though - maybe you need to update the code wherever you are generating these? Eg the [asset_details](https://github.com/dagster-io/dagster/pull/8494/files#diff-6cde5a55291f92ea627f526b8340c0024d40bada5794c6bf15f20bec04a795fc) should have the Lineage tab next to Definition and the [top level the graph](https://github.com/dagster-io/dagster/pull/8494/files#diff-884f6c0498f378e795b5ef4c593b986265ca4a9fa45c25378f99f40a4818c5c5) view doesn't exist anymore.

**Review by shalabhc — CHANGES_REQUESTED** [review]
Added a comment above.

**Comment by erinkcochran87** [comment]
Thanks for pointing that out - I'll take a look!

File: docs/content/tutorial/assets/defining-an-asset.mdx [hunk 1]
```diff
@@ -56,17 +56,17 @@ Serving dagit on http://127.0.0.1:3000 in process 70635
 
 You should be able to navigate to <http://127.0.0.1:3000> in your web browser and view your asset.
 
-<img src="/images/dagster/guides/asset-tutorial/defining-an-asset/defining_an_asset.png" />
+<img src="/images/guides/asset-tutorial/defining_an_asset.png" />
 
 Clicking on the "Materialize All" button will launch a run that will materialize the asset. After that run has completed, the shaded box underneath "cereals" holds information about that run. Clicking on the Run ID, which is the string of characters in the upper right of that box, will take you to a view that includes a structured stream of logs and events that occurred during its execution.
```
Comment (shalabhc) [inline docs/content/tutorial/assets/defining-an-asset.mdx:61]: This text should now be revised to match the new UI. Eg

Click on the "Materialize" button launches a run that materializes the asset. After the run has completed the Activity tab on the assets page holds information about that run. Click on the Run ID in the activity tab to go see details about the run. This opens a view that includes a structured stream of logs and events that occurred during the run execution.

File: docs/content/tutorial/assets/defining-an-asset.mdx [hunk 2]
```diff
@@ -56,17 +56,17 @@ Serving dagit on http://127.0.0.1:3000 in process 70635
...
 
-<img src="/images/dagster/guides/asset-tutorial/defining-an-asset/asset_run.png" />
+<img src="/images/guides/asset-tutorial/asset_run.png" />
 
 In this view, you can filter and search through the logs corresponding to the run that's materializing your asset.
 
 To see a history of all the materializations for your asset, you can navigate to the _Asset Details_ page for it. Click the "cereals" link in the upper left corner of this run page, next to "Success". Another way to get to the same page is to navigate back to the Asset Graph page by clicking "Assets" in the top navigation pane, clicking on your asset, and then clicking on "View in Asset Catalog" at the top of the pane that shows up on the right.
```
Comment (shalabhc) [inline docs/content/tutorial/assets/defining-an-asset.mdx:67]: Revise for new UI. Eg (the activity tab is already shown above)

Click the "cereals" link in the upper left corner of this run page to go back to the asset details page.

**Review by shalabhc — APPROVED** [review]
Screenshots look good! I added a couple of comments about changing the text to match the UI. Approving!

**Pull request merged**